build_doc.sh

```diff
@@ -8,16 +8,47 @@ if [ -z "${KTOR_VERSION}" ]; then
   exit 1
 fi
 
-# Build docs
-versionsDir=$(realpath "./versions/")
-git clone https://github.com/ktorio/ktor/ ktor
+currentVersion=$(cat docs/version.json | jq -r ".version")
+echo "Current docs version is: $currentVersion"
+
+if [ $KTOR_VERSION == $currentVersion ]; then
+  echo "Skipping. The requested version is the same."
+  exit 0
+fi
+
+# Make old versions visible for Dokka
+versionsDir="./versions/"
+if [ -d $versionsDir ]; then
+  echo "Versions directory already exists. Skipping old versions copying."
+else
+  echo "Copying old versions..."
+  mkdir "$versionsDir"
+  cp -R docs ${versionsDir}/${currentVersion}
+  mv ${versionsDir}/${currentVersion}/older/* ${versionsDir}
```

Comment:
Copying the current version and older versions to the `versions/`. This directory will be deleted at the end of the script